docs/root/intro/version_history.rst

```diff
@@ -115,6 +115,7 @@ Version history
 * tracing: the sampling decision is now delegated to the tracers, allowing the tracer to decide when and if
   to use it. For example, if the :ref:`x-b3-sampled <config_http_conn_man_headers_x-b3-sampled>` header
   is supplied with the client request, its value will override any sampling decision made by the Envoy proxy.
+* upstream: fixed crash during host selection from locality weighted clusters with no healthy hosts
```

Comment:
note: generally we haven't been putting bug fixes in the version history, though I don't feel strongly about it one way or the other (if we want to make this a practice in the future we might want to split the release notes into features and bug fix sections).

Reply:
the "this change is user impacting" part of the PR template made me think it should go in, but i'm happy to take it out. happy to take it out if we want the release notes to just be a list of new features

Reply:
I guess for consistency I would probably remove it and we can revisit if we want to start adding bug fixes to the release notes as a community discussion. I will add it to the agenda for the next meeting. Sound good?